src/hotspot/cpu/ppc/vm_version_ppc.cpp

```diff
@@ -109,26 +95,16 @@ void VM_Version::initialize() {
     FLAG_SET_ERGO(TrapBasedRangeChecks, false);
   }
 
-  // Power7 and later.
-  if (PowerArchitecturePPC64 > 6) {
-    if (FLAG_IS_DEFAULT(UsePopCountInstruction)) {
+  if (FLAG_IS_DEFAULT(UsePopCountInstruction)) {
       FLAG_SET_ERGO(UsePopCountInstruction, true);
```

Comment:
Indentation should be adapted. Or even better set the default in globals_ppc.hpp to true and simplify this logic.